extending-valtimo/document/search-fields.md

```diff
@@ -0,0 +1,110 @@
+# Search fields
+
+Search fields add the ability to filter cases. Each search field adds the ability to filter on a specific field.
+This page describes how the search fields can be created, read, updated and deleted.
+
+## Prerequisites
+Only administrators can configure search fields, so the creation, update and deleting of search fields can only be done by administrators.
+
+The retrieval of all available search fields can be done by all logged-in users. 
```

Comment:
Suggestion for writing this sentence in active voice and specifying who or what performed the action

`All logged-in users can do the retrieval of all available search fields. `